templates/google-login-button.php

```diff
@@ -6,18 +6,30 @@
  * @since 1.0.0
  */
 
-$button_text = ( ! empty( $button_text ) ) ? $button_text : __( 'Log in with Google', 'login-with-google' );
+if ( isset( $custom_btn_text ) && $custom_btn_text ) {
+	$button_text = esc_html( $custom_btn_text );
+} else {
+	$button_text = ( ! empty( $button_text ) ) ? $button_text : __( 'Log in with Google', 'login-with-google' );
+}
 
 if ( empty( $login_url ) ) {
 	return;
 }
 
+if ( is_user_logged_in() ) {
+	$button_text = __( 'Log out', 'login-with-google' );
+	$login_url   = wp_logout_url( get_permalink() );
```

Comment:
Variable `$login_url` containing logout URL might become confusing later, instead create another variable here, maybe call it `$button_url` so that we do not override the variables provided to the template. In case this template needs to be extended later this might create confusion and errors.